invite/src/github.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2020 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Octokit} from '@octokit/rest';
+import {GitHubAPI} from 'probot/lib/github';
+
+import {ILogger} from './types';
+
+/**
+ * Interface for working with the GitHub API.
+ */
+export class GitHub {
+  private client: Octokit;
+  private org: string;
+  private logger: ILogger;
+
+  /**
+   * Constructor.
+   */
+  constructor(client: Octokit, org: string, logger: ILogger = console) {
+    Object.assign(this, {client, org, logger});
+  }
+
+  /**
+   * Checks whether a user is a member of the organization.
+   */
+  async userIsMember(username: string): Promise<boolean> {
```

Comment:
Curious - `async` methods still need to declare their return type as `Promise<something>` instead of just `something`?

Reply:
In Typescript, yes